client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content.tsx

```diff
@@ -175,8 +185,21 @@ export default function SiteStatusContent( {
 	}
 
 	if ( type === 'stats' ) {
-		// Content will be added later
-		return null;
+		const { total: totalViews, trend: viewsTrend } = rows.stats.data.views;
+		const trendIcon = getTrendIcon( viewsTrend );
+		return (
+			<span
+				className={ classNames( 'sites-overview__stats-trend', {
+					'is-up': viewsTrend === 'up',
+					'is-down': viewsTrend === 'down',
+				} ) }
+			>
+				<Icon icon={ trendIcon } size={ 16 } />
+				<div className="sites-overview__stats">
+					<ShortenedNumber value={ totalViews } />
+				</div>
+			</span>
+		);
```

Comment:
I need some way to return a gray circle when `viewTrend === 'same'` -- there's no icon for it.

Maybe some logic to show a regular `<svg>` circle instead of an `<Icon>`. I'm open to suggestions.

Reply:
There's no need to use SVG for this, you can achieve this with a few lines of CSS:

```
.circle {
  width: 10px;
  height: 10px;
  border-radius: 50%;
  background-color: gray;
}
```

Reply:
I thought that could be an option, but wanted to ask if it was possible to use the `<Icon>`